decimal.go

```diff
@@ -1601,3 +1601,9 @@ func (d Decimal) Tan() Decimal {
 	}
 	return y
 }
+
+//PiNumber Returns Pi Number For Calculations
+func PiNumber() Decimal {
+	pi, _ := NewFromString("3.14159265358979323846264338327950288419716939937510582097494459")
+	return pi
+}
```

Comment:
As the `PiNumber` function does not provide the capability to set the precision of the number Pi, maybe we should make it a global variable instead of a function?

Reply:
Maybe sending an argument to this function in order to determine the precision wouldn't be a bad idea?

Reply:
I think it would be the best decision here, but it would require more work for sure.